### DISCUSSION TITLE
WHY creating Shovel instead of adding its features to Scoop?

### DISCUSSION THREAD

**Discussion by LoZeno** [Ash258/scoop#192] [Q&A] [answered]
Mostly just curiosity, but what was the driving factor that made you decide to create Shovel as a separate application instead of adding its features to Scoop?

**Comment by Ash258** [comment, marked as answer]
1. Incompetent users who suggest completely nonsense to other users seeking help
2. Stupid requests, which are against basic principles of scoop 
3. Random users, which has no idea what is difference between Scoop and buckets, opening issues in completely unrelated repository.

Not a single reason to keep with the original repository, which has no standards, rules, or even sparkle of the future plans.
Change adoption is too slow and now is scoop already unusable on PowerShell 7.2p8, which means it will be effectively dead in few months after PowerShell 7.2 is released and a newer version of windows will come.

All major changes were done by me with awesome support/help by Rich. Now there is no one who could objectively review, approve and merge changes since Luke and Rich are no longer active.

Look at the Main bucket. After I stopped doing reviews it is unusable and no one takes care of it.

If I need to be the only one who takes care of the project I will rather start with my own fork than waiting XY months for 1 review of PR, which will never come as there is no one who could do the review. Look at zstd extraction for example.
Not to mention that every PR opened to fix zstd extraction is completely unusable and ugly.

Also, all my changes are against luke's conviction of PowerShell verbosity, which effectively goes against his vision.

Now I have full control over everything and I can build lots of other stuff/toolings, which will support Shovel in future. Like the web API and eliminate the need of local git repos in future
